base/src/androidLibMain/kotlin/ContextProvider.kt

```diff
@@ -0,0 +1,17 @@
+package com.splendo.kaluga.base
+
+import android.app.Application
+import android.content.Context
+
+class ContextProvider {
```

Comment:
The implementation I did in https://github.com/splendo/kaluga/issues/29 has some advantages over this one.

- Cannot unset Application, and can only set once.
- Describes it is for holding ApplicationContext, not just any context (could change to Provider)
- Better error messages
- On higher Android version Application is a ContextWrapper, better to return the root context

Also a bit NIH syndrome to redo this?

Reply:
👍